planning/behavior_velocity_planner/include/scene_module/occlusion_spot/manager.hpp

```diff
@@ -45,7 +45,7 @@ class OcclusionSpotModuleManager : public SceneModuleManagerInterface
   const char * getModuleName() override { return "occlusion_spot"; }
 
 private:
-  enum class ModuleID { PRIVATE, PUBLIC };
+  enum class ModuleID { GENERAL, PUBLIC };
```

Comment:
I think ModuleID is better to be simply OCCUPANCY/OBJECT corresponding with method names.

Reply:
changed to occupancy and object
[000cae9](https://github.com/autowarefoundation/autoware.universe/pull/400/commits/000cae9f68725f67d23cbaa7027a0d2d36f2b954)